sqlmesh/core/test/definition.py

```diff
@@ -160,19 +168,67 @@ def create_test(
 
         raise TestError(f"Model '{model_name}' is an unsupported model type for testing at {path}")
 
+    @staticmethod
```

Comment:
[Nit] I see little reason for it to be a class method since it doesn't actually return an instance of the class. I suggest to just make it a function

Reply:
Makes sense